ebean-test/src/test/java/misc/migration/v1_0/EBasic.java

```diff
@@ -41,6 +42,9 @@
 
   @Size(max=127)
   String description;
+  
+  @Column(columnDefinition = "db2;blob(64M);sqlserver,h2;varchar(227);varchar(127)")
+  String description2;
```

Comment:
Please add the same in v1_2. (or only in v1_1) The idea is to check both directions. The models in v1_0 and v1_2 should be equal